R4ML/R/sysml.bridge.R

```diff
@@ -82,10 +82,10 @@ sysml.MatrixCharacteristics <- setRefClass("sysml.MatrixCharacteristics",
 sysml.MLContext <- setRefClass("sysml.MLContext",
   fields = list(env="environment"),
   methods = list(
-    initialize = function(sparkContext = sysmlSparkContext) {
+    initialize = function(sparkContext = r4ml.env$sysmlSparkContext) {
       env <<- new.env()
       if (missing(sparkContext)) {
-        sparkContext = get("sysmlSparkContext", .GlobalEnv)
+        sparkContext <- r4ml.env$sysmlSparkContext
```

Comment:
I think it is good idea to put it in separate object. However I think we have to think of some better name. Since currently r4ml.env is used for user especific setting and also internal settings. we should clearly separate them out. Since r4ml.env is documented in IOP for user specific settings. We should change the internal variable location to may be r4ml.core.env (lets discuss the better name) . Also we can store the pointer r4ml.core.env in the r4ml.env. that way it is more cleaner and keeps us open for other changes that would be important for our user interface